### DISCUSSION TITLE
Add Holidays

### DISCUSSION THREAD

**Discussion by mxmssc** [sql-bi/Bravo#727] [Q&A]
Wondering if/how we can add holidays?
For example in Canada there are civic holidays, Truth and reconciliation, Remembrance Day and they are not included in the current regional dates and I would like to add them.

**Comment by albertospelta** [comment]
Hi @mxmssc, Yes, changing the holidays definition and adding missing ones is definitely possible. You can fork the repository, update the `src/Assets/ManageDates/Templates/HolidaysDefinition.json` and submit a pull request (you can refer to the PR https://github.com/sql-bi/Bravo/pull/695 for an example).

If you prefer, you can also edit the holidays definition directly in PBIDesktop and share your changes here - I will take care of updating Bravo and creating the pull request. You can update the holidays in PBIDesktop by editing the calculated table `HolidaysDefinition`  as shown below. The documentation for the `HolidaysDefinition` template table is available here https://docs.sqlbi.com/dax-template/configuration/definitions/holidays-definition

![image](https://github.com/sql-bi/Bravo/assets/1957788/6b03271c-f235-43b6-9fe1-c841a6236d6b)
